articles/article-117-Why-stick-to-one-example.md

```diff
@@ -0,0 +1,35 @@
+<!-- BEGIN TITLE -->
+# Why stick to one example
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Using the same example, again and again, seems to be quite boring without any added benefit.
+In some instances, however, this can be used for some positive effect. Using a known example to show different things (like methods or algorithms) saves time, as the people don't have to think about the example used in much detail.
```

Comment:
In some cases, a common example can be used for positive effect. Using a known example saves time, since people can draw upon older methods or algorithms to understand new ideas. 
- I am not mentioning that if they don't understand the example then it really does not save time but that is kind of a given

Reply:
Done.